Makefile

```diff
@@ -286,15 +304,13 @@ $(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(APPLICATION_VERSION)-$(TARGET_PLATF
 assets/osx/installer.tiff: assets/osx/installer.png assets/osx/installer@2x.png
 	tiffutil -cathidpicheck $^ -out $@
 
-$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-darwin-$(TARGET_ARCH).dmg: assets/osx/installer.tiff \
-	| $(BUILD_DIRECTORY)
+electron-installer-dmg: assets/osx/installer.tiff
```

Comment:
> Trying to keep the electron-builder configuration in sync with the target name involved a lot of work

But presumably that hard work has been done now, and we won't be changing filenames?

> What would be the benefits of using the actual paths as Makefile targets?

One of the reasons for using a Makefile instead of just a shell script is that `make` can use the dependency-graph to work out which targets need to be rebuilt when other files change, which it obviously can't do if everything is just phoney targets.